interface/forms/CAMOS/new.php

```diff
@@ -1162,47 +1163,47 @@ function processEnter(e,message) {
 <tr>
   <td>
   <div id=id_category_column style="display:inline">
-    <select name=select_category size=<?php echo attr($select_size); ?> onchange="click_category()"></select><br>
+    <select name=select_category size=<?php echo attr($select_size); ?> onchange="click_category()"></select><br />
 <?php
```

Comment:
Todo For a later time: add quotes to all HTML attributes that weren't quoted -- WHY U DO DIS???